src/Sylius/Bundle/ApiBundle/Serializer/ChannelPricingInputDenormalizer.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Sylius\Bundle\ApiBundle\Serializer;
+
+use Sylius\Bundle\ApiBundle\DTO\ChannelPricingInput;
+use Symfony\Component\Serializer\Normalizer\ContextAwareDenormalizerInterface;
+
+class ChannelPricingInputDenormalizer implements ContextAwareDenormalizerInterface
+{
+    public function supportsDenormalization($data, string $type, string $format = null, array $context = [])
+    {
+        return $type === ChannelPricingInput::class;
+    }
+
+    public function denormalize($data, string $type, string $format = null, array $context = [])
+    {
+        return ChannelPricingInput::fromArray($data);
```

Comment:
fromArray method should be deleted from DTO

Reply:
outdated